app/api/events.py

```diff
@@ -262,6 +263,9 @@ def after_create_object(self, event, data, view_kwargs):
         user = User.query.filter_by(id=view_kwargs['user_id']).first()
         uer = UsersEventsRoles(user, event, role)
         save_to_db(uer, 'Event Saved')
+        if data.get('original_image_url'):
+            uploaded_images = create_save_image_sizes(data['original_image_url'], 'event', event.id)
```

Comment:
Also, please download the original_image_url and resave it and update the new location/link